website/screens/principles/typography/TypographyPage.tsx

```diff
@@ -581,9 +582,9 @@ const sections = [
             </Figure>
             <DxcText as="p">
               Styles for the headings can be found in the{" "}
-              <a href="https://developer.dxc.com/design/guidelines/components/heading">
-                component documentation
-              </a>
+              <Link href="/components/heading" passHref>
```

Comment:
We can redirect to `/components/heading/specifications/` since it talks about styles. What do you think?

Reply:
Done!